src/util/cliffThoughtHeight.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Calculate the height that cliff thoughts need to extend.
+ */
+export const calculateCliffThoughtsHeight = ({ deepestDepth, depth }: { deepestDepth?: number; depth?: number }) => {
```

Comment:
This project uses the convention of a single, default export with the same name as the file.

Reply:
Alright, I will factor it out in a `util` function